react/week5/github-usersearch/src/renderusers.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import userContext from "./usercontext";
+import { Link, } from "react-router-dom";
+
+
+function Renderusers() {
```

Comment:
I would rename this component and file to something like `UserList`. It's best practise to name your components based on what they are not what they do. 

All components render, so this title doesn't really provide any further clarification.

Reply:
Actually i already have another component in this app called Userlist. That's why i used renderusers name here. Is there any good name i could give here?!